README.md

```diff
@@ -112,3 +112,17 @@ Options:
 * `--workspace`: Where temporary files will go.  This can be set to your temporary directory.
 * `--end-date-time`: The date that marks the end of the window for which we want to filter the data.
 * `--hours`: The length window in hours for which we want to filter the data.
+
+## Run to download Greek Geological data from University of Athens and take its hash
+
+```bash
+$ cardano-entropy geol-uoa --workspace="$WORKSPACE" --end-date="$END_DATE_TIME'
+Writing to: files/download-geol-uoa-c1ad91d51e90ebb5/all-year.csv
```

Comment:
Should this say `txt` now?

Reply:
Yes after https://github.com/input-output-hk/cardano-entropy/pull/17 having been merged, we will need a new version to reflect this in the binaries unless we want to revert the csv change? 🤷